backend/aurcache/src/builder/docker.rs

```diff
@@ -178,11 +178,27 @@ impl Builder {
 
         let (makepkg_config, makepkg_config_path) =
             create_makepkg_config(name.clone(), build_dir_base)?;
+
+        // Do we need to update everything (paru -Syu) every time?
+        let init_cmd = "
+            sudo pacman-key --init
+            sudo pacman-key --populate archlinux
+            paru -Syu
+        ";
```

Comment:
Arbitrary split between a constant "initialization" part, and the build part that depends on the package. I could merge the two in a single one if desired.